infra-templates/k8s/32/docker-compose.yml.tpl

```diff
@@ -27,6 +27,14 @@ kubelet:
         {{- range $i, $elem := splitPreserveQuotes .Values.ADDITIONAL_KUBELET_FLAGS }}
         - {{ $elem }}
         {{- end }}
+    {{- if ne .Values.HTTP_PROXY "" }}
+    environment:
+        HTTP_PROXY: ${HTTP_PROXY}
```

Comment:
do you think just having the upper cased env vars should be enough? Assuming the convention is : All caps, separated with underscores

Reply:
My bad habit of adding them all because not every software respects `HTTP_PROXY` or `http_proxy` but as we are dealing with one base image and Go components I'll run another test on all caps and update PR.